html/changelogs/.all_changelog.yml

```diff
@@ -40943,9 +40943,9 @@ DO NOT EDIT THIS FILE BY HAND!  AUTOMATICALLY GENERATED BY ss13_genchangelog.py.
 2021-11-04:
   Bokkiewokkie:
   - bugfix: Hopefully made cloning no longer make monochromacy temporary.
-  - bugfix: Shiptest maintenance areas are now no longer named "Hallway".
+  - bugfix: voidcrew maintenance areas are now no longer named "Hallway".
   - bugfix: Allows you to access Syndicate sleepers from the inside again.
-  - code_imp: The dream maker enviroment has been renamed to shiptest.
+  - code_imp: The dream maker enviroment has been renamed to voidcrew.
```

Comment:
Absolutely do not rename shiptest to voidcrew in the changelogs, that's as rude as you get.